crates/c_api/src/frame.rs

```diff
@@ -27,6 +28,10 @@ pub extern "C" fn wasm_frame_func_index(_frame: &wasm_frame_t<'_>) -> u32 {
 ///
 /// This API is unsupported and will panic upon use.
 #[no_mangle]
+#[cfg_attr(
+    feature = "prefix-symbols",
+    export_name = "wasmi_wasm_frame_func_offset"
+)]
```

Comment:
This should use the `#[cfg_attr(feature = "prefix-symbols", wasmi_c_api_macros::prefix_symbol)]` attribute. Probably an oversight due to text replacement without multi-lines.